components/SensorComponent.js

```diff
@@ -12,28 +13,76 @@ const isShaking = (data) => {
 
   // If this force exceeds some threshold, return true, otherwise false
   // Increase this threshold if you need your user to shake harder
-  return totalForce > 1.78;
+  return totalForce > 1.85;
 };
 
 // ==========================
 // = Styled components
 const ShakeView = styled.View`
   display: flex;
   flex-direction: column;
+  align-items: center;
 `;
 
-const ShakeAlert = styled.Text`
-  font-size: 36px;
+const Header = styled.Text`
+  font-size: 25px;
   font-weight: bold;
-  color: #aa0000;
+  color: #b8a51a;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  background: black;
+  padding: 5px;
 `;
-const ShakeDataView = styled.View``;
-const ShakeDataTitle = styled.Text`
+
+const StyledText = styled.Text`
+  margin-top: 70%
+  font-size: 25px;
+  color: #0af2a9;
+  width: 70%
+  text-align: center;
   font-weight: bold;
+ 
+  padding: 5px;
+ 
 `;
-const ShakeData = styled.Text``;
+
+const answers = [
+  "Shit before you shower, then you don't have to wipe",
+  "It is certain",
+  "Without a doubt",
+  "Yes - definitely",
+  "Most likely",
+  "Yes",
+  "No",
+  "Don't count on it",
+  "Ask again later",
+  "Better not tell you now",
+  "My sources say no",
+  "Reply hazy, try again",
+  "Signs point to yes",
+  "I'm sorry what?",
+  "Going into a tunnel...try again",
+  "404 - answer not found",
+  "A communist joke isn't funny... unless everyone gets it",
+  "Perhaps",
+  "I have spoken",
+  "Look into my eyes and it's easy to see. One and one makes two, two and one makes three, it was destiny",
+  "Well that's just like your opinion man",
+  "Where's the money Lebowski?",
+  "Stay out of Malibu!",
+  "That rug really tied the room together",
+  "Tis but a scratch",
+];
 
 export const SensorComponent = () => {
+  const [answer, setAnswer] = useState();
+
+  const updateAnswer = () => {
+    const index = Math.floor(Math.random() * answers.length);
+    setAnswer(answers[index]);
+  };
```

Comment:
My project has another approach so I leared a lot by looking at your way of using an array! Nice job :)